muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/organizationmanagement/rest/OrganizationUserManagementRESTService.java

```diff
@@ -333,6 +339,28 @@ public Response searchStudents(
     SearchResults<List<fi.otavanopisto.muikku.rest.model.Student>> responseStudents = new SearchResults<List<fi.otavanopisto.muikku.rest.model.Student>>(result.getFirstResult(), result.getLastResult(), students, result.getTotalHitCount());
     return Response.ok(responseStudents).build();
   }
+  
+  @GET
+  @Path("/contactPerson")
+  @RESTPermit(OrganizationManagementPermissions.ORGANIZATION_VIEW)
+  public Response listContactPersons() {
+    
+    if (!sessionController.isLoggedIn()) {
+      return Response.status(Status.FORBIDDEN).build();
+    }
+    UserSchoolDataIdentifier userSchoolDataIdentifier = userSchoolDataIdentifierController
+        .findUserSchoolDataIdentifierBySchoolDataIdentifier(sessionController.getLoggedUser());
+    OrganizationEntity organization = userSchoolDataIdentifier.getOrganization();
+    String dataSource = sessionController.getLoggedUserSchoolDataSource();
+    BridgeResponse<List<OrganizationContactPerson>> response = userSchoolDataController.listOrganizationContactPersons(dataSource, organization.getId());
```

Comment:
`organization.getId()` returns OrganizationEntity's id in Muikku's database and it simply cannot be correct here (used as Pyramus' organization.id), pass organization's SchoolDataIdentifier's identifier string instead here.